### PR TITLE
Cpu checkup

### PR DESCRIPTION
**Pull request by Pixmew**
Added a Python script called cpu_usage.py which checks CPU usage in percentage
all Details of usage of file are in scripts comment

### CODE REVIEW

File: cpu_usage.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+#This script will check cpu usage in percentage
```
Comment (subhayu99) [inline cpu_usage.py]: Remove [this file](cpu_usage.py)

**Review by subhayu99 — CHANGES_REQUESTED** [review]
Great work man👌🏻... But, remove the `cpu_usage.py` file from the root directory.

File: cpu_usage.py (hunk 1 above)
Comment (Pixmew) [reply cpu_usage.py]: script is not in root folder it is in advancedscripts

Comment (subhayu99) [reply cpu_usage.py]: Okay... Great✌🏼

**Review by subhayu99 — APPROVED** [review]

**Review by Pixmew — COMMENTED** [review]
all issues should be fixed now

**Comment by subhayu99** [comment]
> all issues should be fixed now

There were some issues with code quality... Although I approved them😅

**Comment by Pixmew** [comment]
all issues are fixed don't worry

**Comment by Pixmew** [comment]
> all issues are fixed don't worry

**Comment by subhayu99** [comment]
> all issues are fixed don't worry

Areee there isn't any wrong in the code execution but rather it wasn't styled according to PEP 8 guidelines and that's why the error was coming.

**Comment by Pixmew** [comment]
That is now fixed yaar

**Comment by subhayu99** [comment]
Yeah u r right brother 🙂

**Pull request merged**